docs/application/web/guides/multimedia/metadata.md

```diff
@@ -44,6 +63,105 @@ When all the needed metadata is extracted, the file handle needs to be released
    fileHandle.release();
    ```
 
+## Extract artwork
+
+1. Create a file handle using the `createFileHandle()` method. Your file needs to have an artwork included.
+
+   ```javascript
+   var filePath = "videos/sample.mp3";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created, artwork can be extracted:
+
+   ```javascript
+   var artwork = fileHandle.getArtwork();
+   ```
+
+3. *Blob* object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(artwork);
+   ```
+
+4. After use, remember to release fileHandle.
+
+## Extract thumbnail frame
+
+1. Create a file handle using the `createFileHandle()` method for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created, thumbnail frame can be extracted:
+
+   ```javascript
+   var thumbnail = fileHandle.getThumbnailFrame();
+   ```
+
+3. *Blob* object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(thumbnail);
+   ```
+
+4. After use, remember to release fileHandle.
+
+## Extract certain frame
+
+1. Create a file handle using the `createFileHandle()` method for a video file.
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created, thumbnail frame can be extracted. You need to provide timestamp information (in milliseconds) and flag indicating to return frame from exact given time (true) or the user can only get the nearest <a href="https://en.wikipedia.org/wiki/Video_compression_picture_types">I-frame</a> of the video (false). Gathering nearest I-frame has better performance.
+
+   ```javascript
+   var timestamp = 2000;
+   var returnExactFrame = true;
+   var frame = fileHandle.getFrameAtTime(timestamp, returnExactFrame);
+   ```
+
+3. *Blob* object can be easily shown in your application in html *&lt;img&gt;* tag
+
+   ```javascript
+   var elem = document.getElementById("blobImage");
+   elem.src = URL.createObjectURL(frame);
+   ```
+
+4. After use, remember to release fileHandle.
+
+## Extract lyrics
+
+1. Create a file handle using the `createFileHandle()` method. Lyrics need to be included in multimedia file as SYLT metadata tag. Supported formats of encoding are at least <a href="http://www.ietf.org/rfc/rfc2279.txt">UTF-8</a> and <a href="http://en.wikipedia.org/wiki/ISO/IEC_8859-1">ISO-8859-1</a>. Other formats can be ignored depending on the platform.
```

Comment:
This part can be added to introductory sentence with a little bit of repharsing:
Lyrics need to be included in multimedia file a...

Reply:
This is a first point of a scenario, I think that it is in correct place, as it describes conditions regarding validity of a file for this scenario.